src/components/AboutPage.jsx

```diff
@@ -0,0 +1,17 @@
+import Header from './Header'
+
+export default function AboutPage(){
+    return(
+        <div className='about-page-container'>
+            <Header/>
+            <div className='about-page-content'>
+                <img style={{width:"100px"}}src="src\assets\img-placeholder.png" alt="" />
```

Comment:
The image is part of the `Header` component if you want to add these attributes there.

Reply:
Added